.github/workflows/_check_clang_static_analyzer.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Clang Static Analyzer
+
+on:
+  workflow_call:
+
+jobs:
+  job_scan_build:
+    name: Clang Static Analyzer
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest
+    steps:
+      - name: Clone
+        uses: actions/checkout@v3
+
+      - name: Build with Clang Static Analyzer
+        run: |
+          make clean
+          scan-build --use-cc=clang -analyze-headers -enable-checker security -enable-checker unix -enable-checker valist -o scan-build --status-bugs make default
```

Comment:
only on nanos?

Reply:
This check would fail on other devices 💀 
IMO we'll need to fix the SDK first to not expose this issue